fe/fe-core/pom.xml

```diff
@@ -42,7 +42,7 @@ under the License.
         <fe_ut_parallel>${env.FE_UT_PARALLEL}</fe_ut_parallel>
         <jacoco.version>0.8.8</jacoco.version>
         <iceberg.version>1.6.0</iceberg.version>
-        <paimon.version>0.8.2</paimon.version>
+        <paimon.version>1.0.0</paimon.version>
```

Comment:
is there any special reason to wait 1.0.1?

Reply:
Not that special, just some optimizations. Can be merged if tests pass.